posixlib/src/main/resources/scala-native/sys/socket.c

```diff
@@ -190,15 +190,19 @@ int scalanative_connect(int socket, struct scalanative_sockaddr *address,
 int scalanative_accept(int socket, struct scalanative_sockaddr *address,
                        socklen_t *address_len) {
     struct sockaddr *converted_address;
-    int convert_result =
-        scalanative_convert_sockaddr(address, &converted_address, address_len);
+    int convert_result = address != null ? // addr and addr_len can be null
```

Comment:
I belive that's the reason why it fails to compile in the CI. Same case below.
```suggestion
    int convert_result = address != NULL ? // addr and addr_len can be null
```